mne/fixes.py

```diff
@@ -1221,6 +1221,287 @@ def __repr__(self):
     # ``if type(self).__module__.startswith('sklearn.')``.
 
 
+###############################################################################
+# Copied from sklearn to simplify code paths
+
+def empirical_covariance(X, assume_centered=False):
+    """Computes the Maximum likelihood covariance estimator
+
+
```

Comment:
extra blank line

Reply:
This is verbatim copy-paste from `sklearn` so I'd rather leave it